paperqa/llms.py

```diff
@@ -278,12 +283,27 @@ async def similarity_search(
 class QdrantVectorStore(VectorStore):
     client: Any = Field(
         default=None,
-        description="Instance of `qdrant_client.QdrantClient`. Defaults to an in-memory instance.",
+        description="Instance of `qdrant_client.AsyncQdrantClient`. Defaults to an in-memory instance.",
     )
     collection_name: str = Field(default_factory=lambda: f"paper-qa-{uuid.uuid4().hex}")
     vector_name: str | None = Field(default=None)
     _point_ids: set[str] | None = None
 
+    def __del__(self):
+        """Cleanup async client connection."""
+        try:
+            loop = asyncio.get_event_loop()
+            if loop.is_running():
+                _ = loop.create_task(self.aclose())  # noqa: RUF006
```

Comment:
What are your thoughts on RUF006, do you mind listening to it?

Also, feel free to tell me you don't think it's worthwhile